submissions/YarHadi/task_js_oop/app.js

```diff
@@ -0,0 +1,133 @@
+const startPlayerX = 200,
+  startPlayerY = 390,
+  cellWidth = 101,
+  cellHeight = 83,
+  leftBorder = 0,
+  rightBorder = 303,
```

Comment:
Is there a correlation betwee cellWidth and this value?
What do we do if we make the field wider (more columns)? Let computer do the maths.
See section 3 of requirements linked by bot for ideas and hints.